index.js

```diff
@@ -30,7 +30,9 @@ export const cancelTx = ({
 export const getRecentSendMessage = async () => {
   let res = await fetch('https://filfox.info/api/v1/message/list?method=Send')
   if (!res.ok) {
-    throw new Error(`Filfox request failed with ${res.status}: ${(await res.text()).trimEnd()}`)
+    const err = new Error(`Filfox request failed with ${res.status}: ${(await res.text()).trimEnd()}`)
+    err.code = 'FILFOX_REQUEST_FAILED'
+    throw err
   }
   const body = await res.json()
```

Comment:
Should we set `err.code` also when `res.json()` throws while reading the response body?

Reply:
We could, but it hasn't happened yet, and might never, so I would like not to spend the extra time on it

Reply:
But happy to review a PR!